x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -697,3 +697,19 @@ c:l| m:i   | m1:i  | c1:l| gender:s
 57 | 25945 | 25945 | 57  | M
 10 | 25324 | 25324 | 10  | null
 ;
+
+
+twoCountStarInStats
```

Comment:
@luigidellaquila You might need to append `#[skip:-8.11.99]` to these two tests since they don't work with 8.11.0 or 8.11.1. Similarly to YAML tests, you can adjust the skipped version after backporting.